src/classes/RecipeRepository.js

```diff
@@ -24,49 +24,58 @@ class RecipeRepository {
   };
 
   getIngredientID(ingName, ingredientsData) {
-    const ingNameLC = ingName.toLowerCase();
-    const ingredientIDFromSearch = ingredientsData.filter((ingredient) => {
-      return ingredient.name === ingNameLC;
-    }).map((ingredient) => {
-      return ingredient.id;
-    });
+    // old version:
+    // const ingNameLC = ingName.toLowerCase();
+    // const ingredientIDFromSearch = ingredientsData.filter((ingredient) => {
+    //   return ingredient.name === ingNameLC;
+    // }).map((ingredient) => {
+    //   return ingredient.id;
+    // });
+    //
+    // return ingredientIDFromSearch
 
+    // refacorted version: returns just the ID number, insted of [ID number]
+    const ingNameLC = ingName.toLowerCase();
```

Comment:
Nice. Good recognition on our need for JUST the id as a number rather than number inside an array. Will be much easier to access/use later.